DotaStats/MainTabBarController.swift

```diff
@@ -6,9 +6,14 @@ final class MainTabBarController: UITabBarController {
         tabBar.backgroundColor = ColorPalette.alternatеBackground
         tabBar.unselectedItemTintColor = ColorPalette.text
         tabBar.tintColor = ColorPalette.accent
+        
+        let coordinator = Coordinator()
+        let network = NetworkServiceImp()
```

Comment:
Лучше писать названия переменных полностью. Тем более, это надо сделать всего 1 раз, автокомплит потом будет тебе помогать
```suggestion
        let networkService = NetworkServiceImp()
```